packages/ocean-lending/src/clients/comptroller.ts

```diff
@@ -4,7 +4,7 @@ import { TxnOptions } from 'options';
 import OToken from './oToken';
 
 export interface OceanMarketSummary {
-  totalLiquidity: number;
+  totalSupplyUSD: number;
```

Comment:
You mean `totalLiquidityUSD` ?

Reply:
My understanding of liquidity is available cash, here meant the total supply.